src/Compilers/CSharp/Test/Symbol/Symbols/CovariantReturnTests.cs

```diff
@@ -39,20 +177,31 @@ public class Derived : Base
 {
     public override string M() => null;
 }
+public class Program
+{
+    void M(Base b, Derived d)
+    {
+        object s1 = b.M();
+        string s2 = d.M();
+    }
+}
 ";
-            var comp = CreateCompilation(source, parseOptions: TestOptions.WithoutCovariantReturns).VerifyDiagnostics(
+            var comp = CreateCompilationWithoutCovariantReturns(source).VerifyDiagnostics(
                 // (8,28): error CS8652: The feature 'covariant returns' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
                 //     public override string M() => null;
                 Diagnostic(ErrorCode.ERR_FeatureInPreview, "M").WithArguments("covariant returns").WithLocation(8, 28)
                 );
             verify(comp);
-            comp = CreateCompilation(source, parseOptions: TestOptions.WithCovariantReturns).VerifyDiagnostics(
+            comp = CreateCompilationWithCovariantReturns(source).VerifyDiagnostics(
                 );
             verify(comp);
+            verify(CompilationReferenceView(comp));
+            verify(MetadataView(comp));
 
             static void verify(CSharpCompilation comp)
             {
                 VerifyOverride(comp, "Derived.M", "System.String Derived.M()", "System.Object Base.M()");
+                VerifyAssignments(comp);
```

Comment:
>VerifyAssignments [](start = 16, length = 17)

It doesn't look like there are assignment expressions in the tree. Is this call going to verify anything?